src/micro/mic_init.f90

```diff
@@ -198,63 +205,63 @@ Subroutine init_ccn (n1,n2,n3,cccnp,cccmp,dn0,ifm)
 implicit none
 
 integer :: n1,n2,n3,i,j,k,ifm
-real, dimension(n1,n2,n3) :: cccnp,cccmp,dn0
-real :: ccn_maxt,ccn_ftt
 real :: ft_hgt ! PJM - height of free tropsophere (boundary layer height + transition depth)
+real, dimension(n1,n2,n3) :: cn1np,cn1mp,dn0
+real :: ccn_maxt,ccn_ftt
```

Comment:
Is there any reason not to update these variables to be "ccn1_maxt" (as in the v6.3.04 main RAMS code update) and ccn1_ftt, here throughout this subroutine? I am trying to avoid future confusion with the new CCN variable names of CCN-1 and CCN-2 instead of just CCN and GCCN.

Reply:
I see an argument to leave these as-is though to be more consistent with what's in the namelist options. Thoughts?

Reply:
I changed ccn_maxt to ccn1_maxt to match the subroutine for ccn2. 

I didn't update ccn_ftt to ccn1_ftt because like you said, it matches the namelist that way. We didn't create a separate CCN_FT or CCN_SH for ccn1 and ccn2 in the namelist. At this point I don't see a reason to separate them out since we're not using CCN2 and having extra entries means more room for mistakes IMO. But I can be convinced otherwise!

Reply:
Sounds good! I think this looks good.